src/web/AmsWebServer.cpp

```diff
@@ -443,6 +479,18 @@ void AmsWebServer::dataJson() {
 	json.createNestedObject("mqtt");
 	json["mqtt"]["lastError"] = (int) mqtt->lastError();
 
+	String domoStatus;
+	if(String(config->getDomoELIDX()).isEmpty()) {
+		domoStatus = "secondary";
+	} else if(mqtt->connected() && config->getMqttPayloadFormat() == 2 && config->getDomoELIDX() > 0) {
+		domoStatus = "success";
+	} else if(mqtt->lastError() == 0) {
+		domoStatus = "warning";
+	} else {
+		domoStatus = "danger";
+	}
+	json["status"]["domo"] = domoStatus;
+
 	serializeJson(json, jsonStr);
```

Comment:
Looks to me like this badge only shows if domoticz is configured or not? If so, is it necessary?

Reply:
Probably not, we can remove it.  I think your idea of moving the domo menue below mqtt and only show  config menue if enabled is good, but I probably need help to implement.

Reply:
Left for now. You can remove if if you want.  Suggest to do it if/when menues are changed.